operator/README.md

```diff
@@ -0,0 +1,147 @@
+# Karmada operator
+
+## Overview
+
+The Karmada operator is a project of the Karmada community designed to manage the lifecycle of Karmada. we can operate CR to install, upgrade and delete the Karmada instance. It is a centrally managed controller that can install karmada instance not only in the local cluster, but also in a remote cluster. It is convenient to manage multiple karmada instances in a global cluster.
```

Comment:
We need to unify the `Karmada instance` and `karmada instance`.

Reply:
I'm with you.